Makefile

```diff
@@ -97,15 +102,27 @@ $(1)-$(2): bindir
 	$(DOCKER_BUILD) 'GOARCH=amd64 GOOS=$(2) $(GO_BUILDCMD) -o $(1)-$(2) cmd/$(3)/main.go'
 ALL_BINS := $(ALL_BINS) $(1)-$(2)
 endef
+
+define BINARY_template
```

Comment:
Let's remove `BINARY` from this name since these templates are working with binaries and it's not specific to this template. Perhaps something like `E2E_OS_template`?

Reply:
yeah, makes sense to me. Fixed!